### PR TITLE
feat(database): create db table Appointments

### PR DESCRIPTION
**Pull request by YasserAmro0**

### CODE REVIEW

File: src/models/appointment.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { DataTypes } from 'sequelize';
+import { AppointmentsAttributes } from '../types/index';
```
Comment (LinaYahya) [inline src/models/appointment.ts]:
```suggestion
import { AppointmentsAttributes } from '../types/';
```

Comment (ahmedmurtaja) [inline src/models/appointment.ts:2]: no need to  type index
```suggestion
import { AppointmentsAttributes } from '../types';
```

File: src/models/appointment.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import sequelize from '../db/connection';
+
+const Appointment = sequelize.define<AppointmentsAttributes>('Appointment', {
```
Comment (LinaYahya) [inline src/models/appointment.ts:5]:
```suggestion
const Appointment = sequelize.define<AppointmentsAttributes>('appointment', {
```

**Review by ahmedmurtaja — APPROVED** [review]

File: src/models/appointment.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  datetime: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  isBooked: {
+    type: DataTypes.BOOLEAN,
```
Comment (LinaYahya) [inline src/models/appointment.ts:16]: let us add a default value as `false`

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by ahmedmurtaja — APPROVED** [review]

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by raghdarizq — APPROVED** [review]

**Review by MohannadSabaa — APPROVED** [review]

**Pull request merged**